tests/m2m_through_regress/tests.py

```diff
@@ -202,7 +203,11 @@ def test_add(self):
 
     def test_add_null(self):
         nullcar = Car.objects.create(make=None)
-        with self.assertRaises(ValueError):
+        with self.assertRaisesMessage(
+            ValueError,
+            '"<Car: None>" needs to have a value for field "car" '
+            'before this many-to-many relationship can be used.'
+        ):
             nullcar.drivers._add_items('car', 'driver', self.unused_driver)
```

Comment:
This error message is confusing. A `Car` instance doesn't have a `car` field. Should it refer to the `make` field instead? The `CarDriver` through table has a `car` foreign key with `to_field='make'`.